### PR TITLE
Sample Bottle Application

### PR DESCRIPTION
**Pull request by salonijain0918**
Sample application for pybrake on bottle framework for swagger specification defined in https://confluence.logicmonitor.com/pages/viewpage.action?pageId=156305152

### CODE REVIEW

File: examples/bottle/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```bash
+python main.py
+```
+
+3. To retrieve the responses, append the endpoints to the localhost URL with a `/`.
+
+Use the below curl commands to interact with the endpoints. The endpoints require an api-key HTTP header.
+
+```bash
+curl "http://localhost:3000/date/" -H 'api-key: b761be830f7c23ebe1c3250d42c43673'
```
Comment (smurf-U) [inline examples/bottle/README.md:30]: @kyrylo  If I'm not mistaken, this `api-key` should be removed from the request. Am I correct?

Comment (kyrylo) [reply examples/bottle/README.md:30]: I think that was our latest idea, but it would be better to verify with @scottsbaldwin.

Comment (scottsbaldwin) [reply examples/bottle/README.md:30]: That's correct. We can omit the api-key for the example application.

**Comment by smurf-U** [comment]
PTAL

File: examples/bottle/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
```
Comment (kyrylo) [inline examples/bottle/README.md:16]:
```suggestion
pip install -r requirements.txt
```

**Review by kyrylo — APPROVED** [review]

**Pull request merged**